### DISCUSSION TITLE
Swaping texture issue on Android

### DISCUSSION THREAD

**Discussion by pnkj1002** [google/model-viewer#4565] [Q&A]
### Description
hi,
I'm using a model with grey texture whose diffuse texture I want to change on model viewer.
I'm changing the texture of the model when load event is fired, which seems to working fine with iOS, but in Android it is not changing the texture. I'm following this documentation.

https://modelviewer.dev/examples/scenegraph/#swapTexturesExample
### Version

- model-viewer: v3.3.0

### Browser Affected

- [x] Chrome, version: 119.0.6045.67
- [ ] Edge
- [x] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [x] SceneViewer
- [ ] QuickLook

### Demo
This is on android
![Screenshot_20231109-153632 1](https://github.com/google/model-viewer/assets/106859150/6ed6ed5c-1e10-42f4-acff-8d1bc8b87fa4)

This is on ios 
![iOs](https://github.com/google/model-viewer/assets/106859150/4443078f-ef77-4b17-95ea-0ac053a8ff05)

**Comment by elalish** [comment]
That whole API doesn't work in Scene Viewer because it redownloads the model from your original URL. You'll need to use WebXR instead if you want this functionality. That's our default ar-mode, so if it's not working, check your JS console - the most common reason to fall back to Scene Viewer is not serving on HTTPS.

**Reply by pnkj1002** [reply]
But if I set mode to WebXR , AR viewer is not opening in Android.

I tried exporting model with textures from ThreeJs, but again it is not opening in android and working fine in iOS. Could be an export issue.
